test/test-trace-api.js

```diff
@@ -94,13 +101,14 @@ describe('Trace Interface', function() {
     
     before(function() {
       traceAPI.enable_(agent);
-      common.init(traceAPI);
-      common.avoidTraceWriterAuth(traceAPI);
+      traceAPI.private_().traceWriter.request = request;
```

Comment:
Even though the functionality is only used in this one file, I had code for doing the above in the `common.js` file so that the only place that uses the `private_()` method directly is the `common.js` file.  I did this because I thought it could make it easier to ensure that the use of `private_()` isn't causing issues with the tests.